src/Components/CreateImageWizardV2/steps/Snapshot/index.tsx

```diff
@@ -13,6 +13,14 @@ export default function SnapshotStep() {
     <Form>
       <Title headingLevel="h1" size="xl">
         Repository snapshot
+        <Alert
+          variant="info"
+          isInline
+          title="The snapshot date only applies to RHEL content."
+        >
+          Third party, custom repositories will always use the newest repository
+          state available.
+        </Alert>
       </Title>
```

Comment:
Can we put the alert after the title instead of inside it? The spacing looks better that way:

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/d34ab08b-d1f2-42e3-a20c-28d70de9bc73)


```suggestion
      <Title headingLevel="h1" size="xl">
        Repository snapshot
      </Title>
      <Alert
        variant="info"
        isInline
        title="The snapshot date only applies to RHEL content"
      >
        Third party, custom repositories will always use the newest repository
        state available.
      </Alert>
```